src/main/java/de/blazemcworld/blazinggames/events/BreakBlockEventListener.java

```diff
@@ -336,4 +351,23 @@ private static void onAnyBlockBreak(Block block) {
             ComputerRegistry.unload(computer.getId());
         }
     }
+
+    private static int getDroppedExp(Block block, ItemStack tool) {
+        if (!(block instanceof CraftBlock craftBlock)) return 0;
+
+        net.minecraft.world.level.block.state.BlockState nmsBlockState = craftBlock.getNMS();
+        BlockPos pos = ((CraftBlock) block).getPosition();
```

Comment:
in the if statement above you defined the `craftBlock` variable, but there's still two casts below
```suggestion
        BlockPos pos = craftBlock.getPosition();
```